### DISCUSSION TITLE
I'm trying to get started ... how do I fetch or post?

### DISCUSSION THREAD

**Discussion by cekvenich2** [mavoweb/mavo#747] [Q&A]
If I want to get data, how do I fetch() an api?

If I want to post some data from a form, how do I save to an api?

**Comment by LeaVerou** [comment]
Hi there,

Mavo is higher level than that, think in terms of what you want to do, not in terms of `fetch()` etc. Did you try going through the [Primer](https://mavo.io/docs/primer)? You will also find the page on [Storage](https://mavo.io/docs/storage) helpful for loading and storing data.

**Comment by cekvenich2** [comment]
Thanks Lea (ps I'm a big fan of you).
I did read the primer but missed storage section, I'll study that so that I can save and get.
